setup/setup.sh

```diff
@@ -126,14 +134,11 @@ installDocker() {
 installDocker
 
 # Pull images:
-sudo docker pull ${DOCKER_PREFIX}/cleanappserver:${DOCKER_LABEL}
```

Comment:
Why that? In GCloud docker commands did not work without sudo.

Reply:
Pulling from DockerHub and GCloud artifact registry is different. I have no clear explanation to that but pulling from inside GCloud doesn'ty require sudo.

Reply:
Sure, I'll add it to README.